deploy/role.yaml

```diff
@@ -88,4 +88,96 @@ rules:
   verbs:
   - '*'
   resources:
-  - credentialsrequests
\ No newline at end of file
+  - credentialsrequests
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
```

Comment:
We have a separate file for cluster roles - https://github.com/integr8ly/cloud-resource-operator/blob/master/deploy/cluster_role.yaml